pkg/kubectl/resource/visitor.go

```diff
@@ -103,8 +106,8 @@ func (i *Info) Visit(fn VisitorFunc) error {
 }
 
 // Get retrieves the object from the Namespace and Name fields
-func (i *Info) Get() error {
-	obj, err := NewHelper(i.Client, i.Mapping).Get(i.Namespace, i.Name)
+func (i *Info) Get() (err error) {
```

Comment:
Keep this as an un-named return value.  You're not using the feature and it often leads to unusual name shadowing.